src/layouts/sidebar-sidecar/utils/product-url-adjusters.ts

```diff
@@ -14,12 +14,16 @@ import remarkPluginAdjustLinkUrls from 'lib/remark-plugin-adjust-link-urls'
  */
 export const vaultUrlAdjuster: Pluggable = [
   remarkPluginAdjustLinkUrls,
-  {
-    urlAdjustFn: (sourceUrl: string): string => {
-      let url = sourceUrl.slice()
-      const isBadApiUrl = url == '/api' || url.startsWith('/api/')
-      if (isBadApiUrl) url = url.replace(/^\/api/, '/api-docs')
-      return url
-    },
-  },
+  { urlAdjustFn: rewriteApiToApiDocs },
 ]
+export const consulUrlAdjuster: Pluggable = [
+  remarkPluginAdjustLinkUrls,
+  { urlAdjustFn: rewriteApiToApiDocs },
```

Comment:
Both `vault` and `consul` share the need to ensure `/api` links get rewritten to `/api-docs` (otherwise we'd have to set up more complex redirects within `dev-portal`). I've re-used the approach from `vault`, and shifted things a bit so it's re-usable for other products as well 👍